packages/volto/theme/themes/pastanaga/collections/form.overrides

```diff
@@ -54,6 +54,10 @@
   }
 }
 
+.ui.form .ui.checkbox {
+  margin: @8px 0;
```

Comment:
`@8px` is a typo or is it a syntax I am not familiar with?

Reply:
That is indeed a syntax you are not familiar with :), it's used all around the pastanaga theme.